### ISSUE TITLE
No such file or directory: stdout.json

### ISSUE THREAD

**Issue by saibotsivad** [MayGo/tockler#21]
### System

Mac OSX El Capitan.
Node: v6.6.0
npm: 3.10.3
Tockler: 3.0.2

### Recreate
- Install Tockler by dragging into Application folder (the one shown in popup when opening dmg file)
- Navigate Finder to Application folder
- Right click on Tockler icon, select 'Open'

### What happens
- The application fails to start and an alert box pops up with the following text:

```txt
A JavaScript error occurred in the main process
Uncaught Exception:
Error: ENOENT: no such file or directory, open '/Users/saibotsivad/Library/Application Support/Tockler/stdout.json'
```

**Comment by saibotsivad** [comment]
After creating the `stdout.json` file, Tockler starts as expected, but an additional (perhaps unrelated) error notification says `Error in auto-updater.`

**Comment by saibotsivad** [comment]
Also, I just tried deleting the `stdout.json` file and restarting Tockler, and it makes the file as I would expect, so perhaps it's something with the initial startup code.

**Comment by MayGo** [comment]
I plan to replace bunyan logger with electron-log(change is made in another branch). Should test with clean run. 

Error in auto updater is fixed in 3.0.3/4. Latest-mac.json was missing in release.

**Comment by saibotsivad** [comment]
I resolved it by installing 3.0.4, thanks! :heart: Note that the README still links to 3.0.2, which is why I installed that one instead of the latest and greatest.
